scalafix-core/src/main/scala/scalafix/util/Patch.scala

```diff
@@ -30,7 +32,16 @@ abstract class ImportPatch(val importer: Importer) extends TreePatch {
 }
 
 object TreePatch {
-  case class Rename(from: Name, to: Name) extends TreePatch
+  trait RenamePatch
+  case class Rename(from: Name, to: Name) extends TreePatch with RenamePatch
+  case class RenameSymbol(from: Symbol, to: Name, normalize: Boolean = false)
```

Comment:
What would the scope of RenameReference be?  Is one renameSyntactic and one renameSemantic?  i still don't clearly understand what rename this instance does. Is it just replacing one single sequence of consecutive tokens?